Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.yml

```
@@ -0,0 +1,128 @@
+category: Database
+commonfields:
+  id: Generic SQL
+  version: -1
+configuration:
+- defaultvalue: MySQL
+  display: SQL DB
+  hidden: false
+  name: dialect
+  options:
+  - MySQL
+  - PostgreSQL
+  - Microsoft SQL Server
+  - Oracle
+  required: true
+  type: 15
+- display: Database host
+  name: host
+  required: true
+  type: 0
+- display: Port
+  hidden: false
+  name: port
+  required: false
+  type: 0
+- display: Database Name
+  hidden: false
+  name: dbname
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: 'Connection Arguments (ex: arg1=val1&arg2=val2)'
+  hidden: false
+  name: connect_parameters
+  required: false
+  type: 0
+description: Generic SQL for the Databases MYSQL, PostgreSQL,Microsoft SQL Server,
```

I think we can remove the names of the DBs we tested on. but we will consult with @kirbles19 regarding this.
```suggestion
description: Generic SQL for the Databases
```